CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ _None_
   [SwiftGen/templates#84](https://github.com/SwiftGen/templates/pull/84)
   [Txai Wieser](https://github.com/txaiwieser)
   [SwiftGen/templates#81](https://github.com/SwiftGen/templates/pull/81)
+* Allow directory to be passed as input of strings parser.
```

Comment:
Please add 2 spaces after the `.`. This is a markdown formatting catch: adding two spaces at the end of a line allows a newline within the same paragraph, so that the next line in your `.md` here will indeed be rendered in a different line once the markdown is rendered, and not the continuation of the current line. That's a strange gotcha of markdown, but that's how it works ;-)

Reply:
humm...I was sure I had done this...thanks

Reply:
Yeah some editors automatically trim trailing spaces so auto-revert those